src/main/java/appeng/me/cache/EnergyGridCache.java

```diff
@@ -59,14 +64,32 @@
 public class EnergyGridCache implements IEnergyGrid
 {
 
+	private static final Comparator<IEnergyGridProvider> COMPARATOR_HIGHEST_AMOUNT_STORED_FIRST = new Comparator<IEnergyGridProvider>()
+	{
+		public int compare( IEnergyGridProvider o1, IEnergyGridProvider o2 )
+		{
+			return Double.compare( o2.getProviderStoredEnergy(), o1.getProviderStoredEnergy() );
+		}
+	};
+
+	private static final Comparator<IEnergyGridProvider> COMPARATOR_LOWEST_PERCENTAGE_FIRST = new Comparator<IEnergyGridProvider>()
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1604)